### PR TITLE
Normalize the root directory state

### PR DESCRIPTION
**Pull request by popzxc**
The following things were done:

- Removed `tox.ini`. Later (when all the tests issues will be fixed) we may add it again and make travis use it instead of plain `python -m unittest` call, but now it useless.
- Removed `Pipfile`. `pipfile` project is not stable yet and not really common. `requirements.txt` were added instead (as a default method to describe dev-dependencies).
- Changelog was updated to match the actual changes that were done (I think from now it makes sense to fill the changelog the same way as in `exonum` core repository).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,25 @@
 # Changelog
 
-### Unreleased
+## Unreleased
 
 - The client has been adopted to work with Exonum dynamic_services branch.
-- Proto files are now downloaded via REST API and compiled dynamically.
+- Proto files are now downloaded via REST API from the Exonum node
+and compiled dynamically.
```
Comment (aleksuss) [inline CHANGELOG.md]: Linter doesn't pass this line. 
```suggestion
  and compiled dynamically.
```

Comment (popzxc) [reply CHANGELOG.md]: Strange, vs code markdown linter didn't show that one :(

Comment (aleksuss) [reply CHANGELOG.md]: At least a linter of the main `exonum` repo 🙃. It would be good to have the same format, wouldn't it ?

Comment (popzxc) [reply CHANGELOG.md]: Totally agree.

**Pull request merged**